source/background.js

```diff
@@ -0,0 +1,5 @@
+const selection = 'Projektteam - zuweisbare Benutzer';
+
+chrome.runtime.onInstalled.addListener(() => {
+	chrome.storage.sync.set({selection});
```

Comment:
why add all the semicolons again?

Reply:
I think it was automatically changed by lint.